ifsbench/launch/mpirunlauncher.py

```diff
@@ -51,10 +50,24 @@ def _get_distribution_options(self, job: Job) -> List[str]:
         ):
             warning('Specified remote distribution option ignored in MpirunLauncher')
 
-        if job.distribute_local is None or job.distribute_local in do_nothing:
+        # By default use core mapping. At least that's the default in the
+        # OpenMPI implementation
+        # (https://docs.open-mpi.org/en/main/man-openmpi/man1/mpirun.1.html#label-schizo-ompi-map-by).
+        map_by = 'core'
+
+        if job.distribute_local and job.distribute_local not in do_nothing:
```

Comment:
I would add `None` to `do_nothing` above and then it should be sufficient to have
```suggestion
        if job.distribute_local not in do_nothing:
```